src/v/cluster/partition_probe.cc

```diff
@@ -168,14 +180,19 @@ void replicated_partition_probe::setup_internal_metrics(const model::ntp& ntp) {
       {sm::shard_label, partition_label});
 
     if (model::is_user_topic(_partition.ntp())) {
+        // Metrics are reported as follows
+        // -2 (default initialized state)
```

Comment:
can we add this to the public description too for downstream systems?

Reply:
done.